wandb/sdk/data_types/base_types/media.py

```diff
@@ -330,7 +336,10 @@ def __eq__(self, other: object) -> bool:
         )
 
     @staticmethod
-    def path_is_reference(path: Optional[str]) -> bool:
+    def path_is_reference(path: Optional[PATH]) -> bool:
```

Comment:
could you just use the typing here, given that it is only used once?
`Union[...]`